backend/project_review_coordination/models.py

```diff
@@ -31,3 +31,28 @@ def save(self, *args, **kwargs):
             # have pk
             self.due_timestamp = timezone.now() + timedelta(hours=1)
         super(ProjectReviewBundleClaim, self).save(*args, **kwargs)
+
+    def get_projects_user_can_review(user):
```

Comment:
is this meant to be a `@staticmethod`?

Just calling the first argument `user` instead of `self` doesn't stop it from being an instance method.

Take a look at `@staticmethod` and `@classmethod` decorators.

Reply:
TIL :smile: